src/cloud/components/Editor/index.tsx

```diff
@@ -454,15 +455,24 @@ const Editor = ({
   )
 
   const setEditorRefContent = useCallback(
-    (newValueOrUpdater: string | ((prevValue: string) => string)) => {
+    (
+      newValueOrUpdater: string | ((prevValue: string) => string),
+      checkboxUpdate = false
```

Comment:
Better rename this. It should explain what it does instead of where it is used.

Reply:
Renamed